src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationService.kt

```diff
@@ -67,12 +67,12 @@ class NotificationService(
         val newShiftNotifications = allPrisons
                 .flatMap { prison ->
                     csrClient.getShiftNotifications(prison.csrPlanUnit, prison.region)
-                }
+                }.distinct().filter { it.actionType != ShiftActionType.EDIT.value }
```

Comment:
👀 didn't you remove this filter a few days ago?

Reply:
no we were mapping them to .Add. I'm adding a comment now.

Reply:
I may be misreading the logic here, but previously we were changing any non duplicate edit entry to an add then remove all the edits. Here it seems like we're removing all the edits even if they are a unique entry (ie they have no corresponding add)